docker-compose.yml

```diff
@@ -17,7 +17,7 @@ services:
       - localstack
 
   localstack:
-    image: localstack/localstack:0.11.3
+    image: localstack/localstack:0.14.4
```

Comment:
On Apple Silicon, my development environment, docker image which has `linux/arm64/v8` arch is preferable.

Reply:
👌